hs/src/Reach/Connector/ALGO.hs

```diff
@@ -3671,6 +3671,7 @@ compile_algo env disp (CPProg {..}) = do
     code "b" [ "apiReturn_noCheck" ]
     label "alloc"
     let ctf f x = do
+          liftIO $ modifyIORef resr $ M.insert (LT.toStrict f) $ AS.Number $ fromIntegral x
```

Comment:
What's going on here?

Reply:
This adds `Global(Bytes/UInt)/Local(Bytes/UInt)` to the `ConnectorObject` result, so the info is accessible when evaluating `new Contract`

Reply:
Ah, I understand